lib/rouge/lexers/jinja.rb

```diff
@@ -96,7 +96,7 @@ def self.word_operators
       end
 
       state :comment do
-        rule(/[^{#]+/m) { token Comment }
+        rule(/[^#]+/m) { token Comment }
```

Comment:
I think this needs a more sophisticated set of rules in this state. This will currently break out of the comment as soon as there's any hash mark which is, I assume, not the correct behaviour.

Can you nest comments in Jinja or does the first `#}` break you out of them?

Reply:
Hmm, I believe the comment structure is `{# ... #}` 

It does appear Jinja added support for inline comments that start with `##` as well, so this probably will require some refactoring

https://jinja.palletsprojects.com/en/2.10.x/templates/#comments
https://jinja.palletsprojects.com/en/2.10.x/templates/#line-statements

Reply:
I've added an extra rule for inline comments and for single `#` characters that are within a commented out section. It looks OK to me but any thoughts?

Reply:
It looks good and works great for me :+1:

Reply:
Great!